code/modules/antagonists/cult/runes.dm

```diff
@@ -841,9 +840,13 @@ structure_check() searches for nearby cultist structures required for the invoca
 	var/turf/T = get_turf(src)
 	var/choice = alert(user,"You tear open a connection to the spirit realm...",,"Summon a Cult Ghost","Ascend as a Dark Spirit","Cancel")
 	if(choice == "Summon a Cult Ghost")
+		var/area/A = get_area(T)
+		if(A.map_name == "Space" || is_mining_level(T.z))
+			to_chat(user, "<span class='cultitalic'><b>The veil is not weak enough here to manifest spirits, you must be on station!</b></span>")
```

Comment:
Not a fan of the use of <b> tags, make a new span and use that.

Reply:
span class='cultbutbold' seems silly